core/jobs/src/main/java/com/cognifide/aet/job/common/comparators/layout/utils/ImageComparison.java

```diff
@@ -79,24 +80,26 @@ public static ImageComparisonResult compare(final BufferedImage pattern, final B
             img1Pixels, img2Pixels);
 
     // Sets not covered areas as error (red)
-    fastMarkOuterAreaAsError(minWidth, minHeight, widthDifference, heightDifference,
+    differenceCounter += fastMarkOuterAreaAsError(minWidth, minHeight, widthDifference, heightDifference,
```

Comment:
i was thinking about saving "canvas-diff" value in separate field but 
what's the reason ?
As far as I know we even don't show differenceCounter  in report ;]